LandSandBoat/server
# TRUST : Create Mumor functionality / enhance samba logic / add basic synergy with Uka Totlihn

**_I affirm:_**

## What does this pull request do?


- The purpose of this PR:

- Mumor: 
    - Adds functionality to Mumor that falls into behaviors described by her BG wiki profile. 
    - Adds synergy modifiers and listeners to manage them dynamically
    - Adds checks for ecosystems (undead) to adjust Samba usage.
    - Adds checks for the players job to adjust Samba usage.
    - [](https://www.bg-wiki.com/ffxi/BGWiki:Trusts#Mumor)

- Uka Totlihn:
    - Minor clean up. 
    - Adds synergy modifiers and listeners to manage them dynamically
    - Corrects teamwork messages.
    - Rearranges gambit order, 
    - Adds checks for ecosystems (undead) to adjust Samba usage.
    - Adds checks for the players job to adjust Samba usage.
    - [](https://www.bg-wiki.com/ffxi/BGWiki:Trusts#Uka_Totlihn)

## Steps to test these changes

1. Load in updated SQL file for Mumor's WS.
2. Restart server to apply.
3. Load in updated Trust luas into the trust directory.
4. Load into the game and summon both trusts.
5. !checklocalvar mumorSynergy to ensure synergy vars are applying.
6. Enter combat and check modifiers WALTZ_POTENTCY and SAMBA_DURATION buffs.
7. Release and / or !hp 0 trusts and !checklocalvar mumorSynergy once despawned that the vars are deleted.

- Mumor Synergy
![image](https://user-images.githubusercontent.com/99501957/195507202-1fa8eafe-7a3a-4a8d-bc24-b9dda865ac48.png)

- Uka Totlihn Synergy
![image](https://user-images.githubusercontent.com/99501957/195507065-668c7f9d-f5a7-4484-bd00-a3b6b8f25e4a.png)
